src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -244,7 +247,7 @@ export const AzureComputeModalBase = ({
       div({ style: { padding: '1.5rem', overflowY: 'auto', flex: 'auto' } }, [
         renderApplicationConfigurationSection(),
         renderComputeProfileSection(),
-        h(PersistentDiskSection, { persistentDiskExists, computeConfig, updateComputeConfig, setViewMode, tool }),
+        h(PersistentDiskSection, { persistentDiskExists, computeConfig, updateComputeConfig, setViewMode, cloudPlatform: cloudProviderTypes.AZURE }),
```

Comment:
TOL: In the future work (with .ts conversion) we'll likely want to narrow down what we pass to PersistentDiskSection.

Thinking we should manage disk config state separately from compute config, and then merge at submission time.